components/brave_rewards/resources/page/reducers/rewards_reducer.ts

```diff
@@ -252,6 +249,11 @@ const rewardsReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State
         break
       }
 
+      if (!enabled) {
+        state.balance = defaultState.balance
+        state.promotions = []
+      }
```

Comment:
Do we need this in the android page as well?

Reply:
added, even though that design is different, but just to be consistent